sdk/eventgrid/azure-messaging-eventgrid/src/main/java/com/azure/messaging/eventgrid/implementation/EventGridPublisherClientImpl.java

```diff
@@ -136,7 +136,7 @@ public SerializerAdapter getSerializerAdapter() {
                 @BodyParam("application/cloudevents-batch+json; charset=utf-8") List<CloudEvent> events,
                 Context context);
 
-        @Post("")
+        @Post("/api/events")
```

Comment:
EventGrid takes the passed-in endpoint instead of parsing the endpoint into host and paths. Is it possible to customize code generator?